data/sql_incremental_aggregates/prepare_schedule_a_aggregate_employer.sql

```diff
@@ -14,60 +14,17 @@ and is_individual(contb_receipt_amt, receipt_tp, line_num, memo_cd, memo_text, c
 group by cmte_id, cycle, employer
 ;
 
-alter table ofec_sched_a_aggregate_employer_tmp add column idx serial primary key;
+alter table ofec_sched_a_aggregate_employer_tmp add column idx serial;
```

Comment:
I'm going to use this script to capture most notes for all of the aggregate scripts, since they'll all be identical. :-)

Noting here to just double check on whether or not we need a unique identifier field for materialized views built off of these aggregate tables based on our conversation with @LindsayYoung from earlier!

Also, we do have models in the Python code that are built with all of these tables (found in [webservices/common/models/aggregates.py](https://github.com/18F/openFEC/blob/develop/webservices/common/models/aggregates.py) and those are all built off of the [`BaseModel`](https://github.com/18F/openFEC/blob/develop/webservices/common/models/base.py#L56), which references an `idx` field.  So if we are able to go down this route of not needing an arbitrary unique identifier field, we'd have to change some of the application code as well.

Reply:
@LindsayYoung, can you point me to the code where the materialized views need the `idx` column?

Reply:
@ccostino, you're right, we would remove `BaseModel` as a parent for these models.

Reply:
I was thinking of two constraints, but they don't apply in this case.

These are tables, I was thinking they were materialized views. To refresh materialized views concurrently there needs to be at least one unique key constraint. https://www.postgresql.org/docs/9.4/static/sql-refreshmaterializedview.html

The other consideration was that sqlalchemy needs a unique primary key or unique composites for all its models. We could make composites for the aggregates, but for most of the data I would not trust things to be unique.

So, this looks good. Thanks!

Reply:
Just to clarify, we are making composite unique keys with the columns that are being aggregated on.

Reply:
Awesome!  So it sounds like we can move forward with just the composite keys and update the models referencing the aggregate tables to use those instead (and not inherit from `BaseModel`)?

Reply:
Since the `idx` column is being kept in all cases here, we don't need to adjust the `BaseModel` inheritance, correct?